PHASE_1/API_SourceCode/src/utils/Helpers.ts

```diff
@@ -18,3 +22,14 @@ export const parseHeaders = (
     keyTerms,
   };
 };
+
+export const parseReportHeaders = (
```

Comment:
we don't need this function since we aren't getting any specific information from the headers for these requests